app/services/foods_service.rb

```diff
@@ -0,0 +1,39 @@
+class FoodsService
+
+  def self.search_ingredients(ingredient)
+    paramiters = {query: ingredient}
+    response = connection.get('fdc/v1/foods/search') do |faraday|
+      paramiters.each do |k, v|
+        faraday.params[k] = v
+      end
+    end
```

Comment:
This api also can take an optional parameter of `per_page` which you could leverage to get only 10 foods back since the default is to just send the first page.